configure-buildfarm/tasks/main.yml

```diff
@@ -48,6 +48,19 @@
     - config
     - security
 
+- name: "Add SSH public key to the user configuration page automatically"
+  uri:
+    url: "{{ jenkins_route_protocol | default('https') }}://{{ route_output.stdout }}/user/{{ jenkins_username | default('admin') }}/configSubmit"
+    method: POST
+    force_basic_auth: true
+    # 'fullName' and 'description' must be specified
+    body: "json={\"fullName\": \"Jenkins Admin\", \"description\": \"\", \"userProperty10\": {\"authorizedKeys\": \"{{ public_key.stdout | urlencode }}\"}}"
+    validate_certs: no
+    user: admin
+    password: password
```

Comment:
Would it be possible to retrieve the admin password from the jenkins dc object?

Reply:
We can use just same value as in template: https://github.com/aerogear/aerogear-digger-installer/blob/master/deploy-jenkins/templates/jenkins-persistent-template.j2#L329
But sourcing password from jenkins/dc is more reliable.